docs/get-started/python.md

```diff
@@ -0,0 +1,82 @@
+---
+parent: Get started
+---
+
+# Build SDKs for Python
+
+## Required tools
+
+- [Python 3.10.4](https://www.python.org/)
+- [pip 20.0.2](https://pip.pypa.io/en/stable/)
+
+## Target project requirements
+
+Before you can compile and run the target project, you will need to initialize it. After initializing the test project, you will need to add references to the [abstraction](https://github.com/microsoft/kiota-abstractions-python), [authentication](https://github.com/microsoft/kiota-authentication-azure-python), [http](https://github.com/microsoft/kiota-http-python), [serialization JSON](https://github.com/microsoft/kiota-serialization-json-python), and [serialization Text](https://github.com/microsoft/kiota-serialization-text-python) packages.
+
+## Creating target projects
+
+> **Note:** you can use an existing project if you have one, in that case, you can skip the following section.
+
+Execute the following commands in the directory where you want to create a new project.
+
+```bash
+py someCommand
```

Comment:
What is this `py`? It doesn't work on my machine. Is it an alias for python or python3? We should change these commands to something that will work for any environment with just the pre-reqs.

Reply:
Also updated in https://github.com/microsoft/kiota/pull/1792/commits/3b79f65263a4910ff6329073bfa23b4b1dd5ff7a